Subread/2.0.0/FeatureCounts.nf

```diff
@@ -0,0 +1,40 @@
+process FeatureCounts {
+    tag {"Subread FeatureCounts ${run_id}"}
+    label 'Subread_2_0_0'
+    label 'Subread_2_0_0_FeatureCounts'
+    container = 'quay.io/biocontainers/subread:2.0.0--hed695b0_0'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+        val(run_id)
+        path(bam_file)
```

Comment:
Verwacht je niet een tuple met sample, bam en bai?

Reply:
Hi/Zij verwacht een bam file of een collectie van bam files. FeatureCounts werkt in dit geval op run en niet sample niveau.

Reply:
Ah okay, dan zou ik er `bam_files` van maken zodat je laat zien dat er 1 of meer bam files als input gebruikt kunnen worden.